# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/kasc/squaroid/gcc34.patch
# ROCK Linux is Copyright (C) 1998 - 2006 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./src/menu.c.orig	2004-09-12 08:38:43.843375072 +0200
+++ ./src/menu.c	2004-09-12 08:39:11.352193096 +0200
@@ -132,36 +132,36 @@
   return gtk_item_factory_get_widget (item_factory, "<main>");
 }
 
-static char *help_commands = "
-  Squaroid Help
-  == == == == == == =
-  Available chatroom commands
-  * /exit, /quit, /part
-  Disconnect from server, and reopen the Connect Dialog
-  * /disconnect
-  Disconnect from server, and DO NOT reopen the Connect Dialog
-  * /score < nickname >
-  Get the scores for player called < nickname >
-  */allscores
-  Get scores for all players currently on this channel.
-  * /solo Start a solo game.This is one of three ways to start a solo game;)
-  * /msg < nickname > <message > |/notice < nickname > <message >
-    Send a private < message > to < nickname >
-    The score commands only work if the SquaroidHost bot
+static char *help_commands = "\
+  Squaroid Help\
+  == == == == == == =\
+  Available chatroom commands\
+  * /exit, /quit, /part\
+  Disconnect from server, and reopen the Connect Dialog\
+  * /disconnect\
+  Disconnect from server, and DO NOT reopen the Connect Dialog\
+  * /score < nickname >\
+  Get the scores for player called < nickname >\
+  */allscores\
+  Get scores for all players currently on this channel.\
+  * /solo Start a solo game.This is one of three ways to start a solo game;)\
+  * /msg < nickname > <message > |/notice < nickname > <message >\
+    Send a private < message > to < nickname >\
+    The score commands only work if the SquaroidHost bot\
     is also logged in with the nickname `Squaroid'.";
 
-static char *help_options = "
-  Available Options
-  * Skip MOTD
-  Skip the Message Of The Day that each IRC server sends
-  you.
-  * Auto Decline
-  This automatically declines every game challenge you
-  might receive.I.e.if you are playing a game and
-do
-    not
-    want to be disturbed this option is for you
-    * Save Settings on Exit
+static char *help_options = "\
+  Available Options\
+  * Skip MOTD\
+  Skip the Message Of The Day that each IRC server sends\
+  you.\
+  * Auto Decline\
+  This automatically declines every game challenge you\
+  might receive.I.e.if you are playing a game and\
+do\
+    not\
+    want to be disturbed this option is for you\
+    * Save Settings on Exit\
     I think this is self explanatory ! ";  
     void menucallback_help (GtkWidget * w, gpointer data)
     {
